spec/features/tasks_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'feature_helper'
+
+describe "Tasks" do
```

Comment:
I think it could be great! However, all of our specs are currently using this same syntax and the `:feature` [tag is being added automatically](https://relishapp.com/rspec/rspec-rails/docs/feature-specs/feature-spec) since our feature specs are being placed under `specs/features`.